packages/s2-core/src/utils/export/method.ts

```diff
@@ -55,15 +44,29 @@ export function getAllLevels(interactedCells: (RowCell | ColCell)[]) {
 }
 
 /**
- * 复制/导出时会在文本的两侧中增加制表符，用于在 Excel 中展示
- * 兼容极端情况，防止维值中本身就存在制表符的情况，如：“成都市\t” => "成都市\t\t" 导致错列
+ * https://en.wikipedia.org/wiki/Comma-separated_values#Example
+ * 根据 CSV 规范，按以下规则处理字段内容：
+ * 若字段包含 ,、"、\n 或 \t → 用双引号包裹字段。
+ * 若字段中的双引号 → 转义为两个双引号 ""。
+ * @param field
  */
-export const trimTabSeparator = (text: string) => {
-  if (!includes(text, TAB_SEPARATOR)) {
-    return text;
+export const escapeField = (field: SimpleData): SimpleData => {
+  if (typeof field !== 'string') {
+    return field;
+  }
+
+  // 检查是否需要转义：包含逗号、双引号或换行符
+  if (/[",\r\n\t]/.test(field)) {
+    // 转义双引号 -> 两个双引号
+    field = field.replace(/"/g, '""');
```

Comment:
直接链式 replace 吧，另外尽量不直接修改形参

```suggestion
    const newField = field.replace(/"/g, '""').replace(/(?<!\r)\n/g, '\r\n');
    
    return `"${newField}"`;
```